airbyte-webapp/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+import styled from "styled-components";
+
+const Container = styled.div`
+  * {
+    color: rgba(59, 69, 78, 1);
+    line-height: 20px;
+    font-weight: 400;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 56px;
+  }
+
+  a {
```

Comment:
react-markdown has `components` prop, I think we should reuse it and pass current stylings.